### DISCUSSION TITLE
How can I disable or move Borg Backup?

### DISCUSSION THREAD

**Discussion by RedFoxy** [nextcloud/all-in-one#2262] [Questions] [answered]
Hi!
I've Borg Backup active on my NextCloud AIO, but I need to move it in a new directory, there is a way to move it or disable/delete and than create it again in another directory?

Thank you!

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/596

**Reply by RedFoxy** [reply]
So there isn't a way to delete the repository to start again?

**Reply by szaimen** [reply]
You asked for either or.

As for deleting, you would need to delete borg related files in the data directory of the mastercontainer volume, adjust the path in the configuration.json, possibly more
